applications/plugins/SofaMatrix/CMakeLists.txt

```diff
@@ -7,7 +7,7 @@ sofa_find_package(Sofa.Component.Constraint.Lagrangian.Solver REQUIRED)
 sofa_find_package(Sofa.Component.LinearSolver.Direct REQUIRED)
 sofa_find_package(CImgPlugin REQUIRED)
 sofa_find_package(Eigen3 REQUIRED)
-sofa_find_package(SofaGuiQt REQUIRED QUIET)
+sofa_find_package(SofaGuiQt QUIET)
```

Comment:
```suggestion
sofa_find_package(Sofa.GUI.Qt QUIET)
```

Reply:
and I cannot suggest non-modified lines but you would need to change all the following references to `SofaGuiQt` afterwards, with `Sofa.GUI.Qt`